include/swift/SIL/AbstractionPattern.h

```diff
@@ -679,7 +679,7 @@ class AbstractionPattern {
   /// imported as a method.
   ///
   /// For example, if the original function is:
-  ///   void CCRefrigatorSetTemperature(CCRefrigeratorRef fridge,
+  ///   void CCRefrigeratorSetTemperature(CCRefrigeratorRef fridge,
   ///                                   CCRefrigeratorCompartment compartment,
   ///                                   CCTemperature temperature);
```

Comment:
I haven't tried to adjust indentation. I can, but I'd rather wait for a general direction before I do this.